### ISSUE TITLE
commoncode breaks with click 8.0.1

### ISSUE THREAD

**Issue by priv-kweihmann** [nexB/commoncode#18]
```
  File "/usr/lib/python3.9/site-packages/commoncode/cliutils.py", line 373, in __init__
    super(PluggableCommandLineOption, self).__init__(
  File "/usr/lib/python3.9/site-packages/click/core.py", line 2482, in __init__
    super().__init__(param_decls, type=type, multiple=multiple, **attrs)
  File "/usr/lib/python3.9/site-packages/click/core.py", line 2041, in __init__
    self.type = types.convert_type(type, default)
  File "/usr/lib/python3.9/site-packages/click/types.py", line 1019, in convert_type
    return FuncParamType(ty)
  File "/usr/lib/python3.9/site-packages/click/types.py", line 158, in __init__
    self.name = func.__name__
AttributeError: 'str' object has no attribute '__name__'
```

FYI

**Comment by pombredanne** [comment]
@priv-kweihmann sorry for the reply as I missed this... we now support alright this version of click
This should have been fixed with https://github.com/nexB/commoncode/commit/02fb544869708607997bbf3440e1b402c68a3164 and was also related to https://github.com/nexB/scancode-toolkit/issues/2529

Can you confirm that this is working now?

**Comment by pombredanne** [comment]
I confirm that this should be all fixed in ScanCode TK v21.6.7 and CommonCode v21.5.5
Thanks for the report!

**Comment by priv-kweihmann** [comment]
> @priv-kweihmann sorry for the reply as I missed this... we now support alright this version of click
> This should have been fixed with [02fb544](https://github.com/nexB/commoncode/commit/02fb544869708607997bbf3440e1b402c68a3164) and was also related to [nexB/scancode-toolkit#2529](https://github.com/nexB/scancode-toolkit/issues/2529)
> 
> Can you confirm that this is working now?

@pombredanne I would if there would be a "non pre release" version of scancode-toolkit :-)
Any timeline when to expect that?

**Comment by pombredanne** [comment]
ScanCode v21.6.7 pre-release status has been removed. In earnest this was an experiment to treat a new release as some sorts of "pre-release" for a short while, yet given the processes in places and the continuous QA that is taking place, this serves no good purpose. (NB: there is a new release os SCTK coming up anytime now too)
